tests/nightly/apache_rat_license_check/rat-excludes

```diff
@@ -49,3 +49,4 @@ include/*
 .*.json.ref
 searchtools_custom.js
 theme.conf
+LICENSE.binary.dependencies
```

Comment:
how about we generate this file when building the packages?

Reply:
The License may change through time and need a lot of checking before we use. Most of the content inside are hand-craft which are hard to automate. I would recommend to do this manually.